### DISCUSSION TITLE
How to lock a discussion category?

### DISCUSSION THREAD

**Discussion by aliok** [community/community#57008] [Discussions]
### Select Topic Area

Product Feedback

### Body

We use GitHub discussions in our repository, https://github.com/cncf/mentoring/discussions, where we create a new category for each mentoring season/term. Once the term is over, we would like to lock the category for the specific term.

Any good way of doing this? 

Also, we would need to "archive" old categories too. We want the old discussions to be in their categories, but we don't want to show the old categories on the main discussions page.

**Comment by LindseyB** [comment]
Hi - sorry this isn't currently possible. Right now the suggestion would be to mark all the discussions in that category as closed, but I understand that's not the best flow. The graphql API or a github action could be the right course of action of automating this.

**Reply by aliok** [reply]
@LindseyB How can I report this as a feature request to GitHub?

**Reply by LindseyB** [reply]
Just putting it in this forum is enough for us to know.
